### ISSUE TITLE
[Table]: Support boolean "fixed" prop

### ISSUE THREAD

**Issue by thien-do** [moaijs/moai#116]
After @ZeroX-DG 's great PR #110 , now Table supports fixed last column. However, the new "fixed" type is an object, which is quite incompatible with the previous version, especially when we highly recommend strict TypeScript. To avoid a breaking change with current consumers, we should support the option to use "fixed" as a boolean. I think this is simple and would be a nice compromise toward 1.0.

Basically, the type of the "fixed" prop should be

```
fixed: TableFixed | boolean
```

and later we have a small check: if the prop is "false", then do nothing, if it's "true", then judge it as { header: true, first: true }

**Comment by thien-do** [comment]
Actually we're going to cut 1.0 so let just move on with the better, explicit object-based API
